scss/login-detail.scss

```diff
@@ -0,0 +1,78 @@
+.login-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    height: 400px;
+    width: 300px;
+    background-color: $wild-sand;
+    padding-bottom: 24px;
+    border-top: 8px solid $cerulean;
+  
+    .wolox-logo{
+      align-self: center;
+      height: 70px;
+      width: 212px;
+    }
+  
+    .text{
+      align-self: flex-start;
+      color: $black;
+      font-size: 16px;
+      line-height: 24px;
+      margin-left: 24px;
+    }
+  
+    .credentials-box{
+      height: 32px;
+      width: 252px;
+      border-radius: 5px;
+      background-color: $white;
+    }
+  
+    .button{
+  
+      .login-button{
```

Comment:
Check the invision, it seems that there are 2 types of buttons: primary and secondary for example. It will prove worthwhile to define it's classes so it can be reused by any view that needs them